frontend/controller/actions/group.js

```diff
@@ -173,7 +173,10 @@ export default (sbp('sbp/selectors/register', {
             permissions: [GIMessage.OP_KEY_REQUEST],
             meta: {
               quantity: 60,
-              expires: Date.now() + DAYS_MILLIS * INVITE_EXPIRES_IN_DAYS.ON_BOARDING,
```

Comment:
See my comment for how _not_ to set an expiry date (i.e., leave this blank). While you're at it, I'd also use the server time where possible (`sbp('chelonia/time')`)